internal/crdReplicator/env_test.go

```diff
@@ -53,6 +57,12 @@ func setupEnv() {
 		panic(err)
 	}
 	dynClient = dynamic.NewForConfigOrDie(configLocal)
+	dynFac = dynamicinformer.NewFilteredDynamicSharedInformerFactory(dynClient, 30*time.Second, metav1.NamespaceAll, func(options *metav1.ListOptions) {
```

Comment:
I think it can be useful to parameterize the `resyncPeriod`. Maybe we could use a flag to enable the customization of such parameter?

Reply:
it was already parametrized, so it's now used where needed. The `resyncPeriod` value is used to periodically check the watched resource in case if any of the events was last due to network problems or timeouts. I don't think it could be of great use for the user to be able to set it.